sdk/core/azure-core/test/ut/service_tracing_test.cpp

```diff
@@ -88,6 +88,28 @@ TEST(TracingContextFactory, SimpleServiceSpanTests)
     EXPECT_FALSE(contextAndSpan.Context.IsCancelled());
   }
 }
+
+TEST(TracingContextFactory, DeprecatedFactoryCtorForServiceWhichReleasedWithThisDependency)
+{
+  Azure::Core::_internal::ClientOptions clientOptions;
+#ifdef _MSC_VER
+#pragma warning(push)
```

Comment:
Can we also remove the CMakeFile-wide warning disabling, and only disable them on a tactical level as necessary, the same way that you're doing here? - https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core-tracing-opentelemetry/test/ut/CMakeLists.txt#L12-L19

Reply:
Already done. That was a last gasp attempt at trying to deal with the problem.